app/event/[id]/page.tsx

```diff
@@ -0,0 +1,106 @@
+const comments = [
+  {
+    id: 1,
+    author: "alice.bsky.social",
+    content: "looking forward to this! working on a new **react** project",
+  },
+  {
+    id: 2,
+    author: "bob.bsky.social",
+    content: "see you there ☕",
+  },
+];
+
+export default async function EventPage({
+  params,
+}: {
+  params: Promise<{ id: string }>;
+}) {
+  const { id } = await params;
+
+  return (
+    <div className="space-y-8">
+      <div>
+        <div className="flex items-start justify-between pb-2">
+          <h1 className="text-xl">code & coffee #42</h1>
+          <a
+            href={`/event/${id}/edit`}
+            className="text-sm text-orange-600 hover:underline"
+          >
+            edit
+          </a>
+        </div>
+        <div className="pb-4 text-sm text-gray-600">
+          tomorrow 9:00am • local cafe
+        </div>
+        <div className="border-b border-gray-200"></div>
+      </div>
+      <div>
+        <h2 className="pb-2">description</h2>
+        <div className="space-y-2 text-gray-700">
+          <p>
+            join us for our weekly code & coffee session. bring your laptop,
+            grab a coffee, and work on projects together.
+          </p>
+          <p>perfect for:</p>
+          <ul className="list-inside list-disc pl-4">
+            <li>networking with local developers</li>
+            <li>getting help with coding problems</li>
+            <li>finding collaborators</li>
+          </ul>
+        </div>
+      </div>
+      <div>
+        <h2 className="pb-2">location</h2>
+        <p className="text-gray-700">
+          local cafe
+          <br />
+          123 main st
+          <br />
+          anytown, usa
+        </p>
```

Comment:
using `<br />` seems like a bad idea, perhaps we can do something like this? (especially if that data we eventually use includes newlines.. but may be worth experimenting with a bit)

```suggestion
        <p className="text-gray-700 whitespace-pre-line">
          local cafe
          123 main st
          anytown, usa
        </p>
```